.github/workflows/ci-sanitizers.yml

```diff
@@ -0,0 +1,128 @@
+name: CI with sanitizers enabled
```

Comment:
>CI with sanitizers enabled [](http://example.com/codeflow?start=6&length=26)

As commented by Ahmed in discussion, can we instead include this into the existing CI workflow? Rather than creating a separate workflow?

Reply:
Build in debug mode for sanitizer is a good argument to keep this a separate workflow. Thanks for explaining!